src/url-loader/fs-url-loader.ts

```diff
@@ -29,13 +29,13 @@ export class FSUrlLoader implements UrlLoader {
   root: string;
 
   constructor(root?: string) {
-    this.root = root || '';
+    this.root = usePathSep(root || '');
```

Comment:
Not saying that relative/unresolved paths are recommended, but people do that stuff all day long.